v2/session/common-customizations/sessions/revoke-session.mdx

```diff
@@ -111,4 +110,43 @@ app.use("/revoke-all-user-sessions", async (req, res) => {
 });
 ```
 </TabItem>
-</BackendSDKTabs>
\ No newline at end of file
+</BackendSDKTabs>
+
+## Access token blacklisting
+
+Once this feature is enabled, each session verification attempt will result in a database call. This will make sure that if session revocation has taken place, access tokens on the front end will be invalidated.
+
+:::caution
+Note that enabling this feature may slow down all of your API calls which require session verification.
+:::
+
+For example, in the APIs defined in [Method 2](#method-2-revoking-a-session-using-a-sessionhandle), [Method 3](#method-3-revoking-multiple-sessions-using-an-array-of-sessionhandle) and [Method 4](#method-4-revoking-all-sessions-for-a-userid), if the access token blacklisting feature is enabled, these APIs will now result in the access token being cleared from the frontend.
```

Comment:
- Why have you put "for example"?
- It's not that doing those methods will clear it on the frontend. It's that even if the tokens exist on the frontend, it won't matter cause session verification will require a db call each time - and those will fail